Consistent identifier formatting in the document (#649)
VDA5050/VDA5050: VDA5050_EN.md

Reviewer: loads array exists further down the list
```suggestion
```

Author: `loads` vs. `load`: We had a discussion if only adding an object to the loads array triggers the state message or if changes within the individual load objects trigger the message. We could combine it. What do you think?

Author: Done directly in PR

- Change in the `nodeStates` or `edgeStates` arrays
- Change in the `actionStates`, `instantActionStates` or `zoneActionStates` arrays
- Change in the `zoneSets` array

> - Changes in the `loads` array

Author: ```suggestion
- Changes in the `loads` array or the individual `load` objects
```

Author: Removed, not necessary

Reviewer: + " Change in the `actionStates`, `instantActionStates` or `zoneActionStates` arrays or the individual `actionState` objects
+ Change in the `edgeRequests` or `zoneRequests` arrays or the individual `request` objects
... and so on. I would leave all of that out as it should be clear

Author: Yes had the same thought. We added the remark in #574.

| **Field** | **data type** | **description** |
|---|---|---|
| *loadPositions* | array of string | Array of load positions / load handling devices.<br>This array contains the valid values for the parameter `state.loads[].loadPosition` and for the action parameter `loadHandlingDevice` of the actions `pick` and `drop`.<br>*If this array does not exist or is empty, the mobile robot has no load handling device.* |
| ***loadSets [loadSet]*** | array | Array of load sets that can be handled by the mobile robot |
| { | | |
|&emsp; setName | string | Unique name of the load set, e.g., DEFAULT, SET1, etc. |
|&emsp; loadType | string | Type of load, e.g., EPAL, XLT1200, etc. |
|&emsp; *loadPositions* | array of string | Array of load positions or load handling devices, this load set is valid for.<br>*If this parameter does not exist or is empty, this load set is valid for all load handling devices on this mobile robot.* |
|&emsp; ***boundingBoxReference*** | JSON object | Bounding box reference as defined in the `loads` array in the `state` message. |

> |&emsp; ***loadDimensions*** | JSON object | Load dimensions as defined in the `loads` array in the `state` message. |

Reviewer: Not defined in the state message itself, but the state message chapter
```suggestion
|&emsp; ***boundingBoxReference*** | JSON object | Bounding box reference as defined in [7.8 Implementation of the state message](#78-implementation-of-the-state-message). |
|&emsp; ***loadDimensions*** | JSON object | Load dimensions as defined in [7.8 Implementation of the state message](#78-implementation-of-the-state-message). |
```
